packages/block-library/src/read-more/index.php

```diff
@@ -19,15 +19,22 @@ function render_block_core_read_more( $attributes, $content, $block ) {
 	}
 
 	$post_ID            = $block->context['postId'];
+	$post_title         = get_the_title( $post_ID );
+	$screen_reader_text = sprintf(
+		/* translators: %s is either the post title or post ID to describe the link for screen readers. */
+		__( ': %s' ),
+		'' !== $post_title ? $post_title : __( 'untitled post ' ) . $post_ID
```

Comment:
We received this feedback related to this change https://core.trac.wordpress.org/timeline?from=2023-02-07T07%3A30%3A28Z&precision=second

Basically we should avoid concatenation when it comes to translated strings. 

Let's make sure to follow-up on that and label the PR with the "backport to WP beta/RC" label to include on the next package update in core.

Reply:
Tracked here https://github.com/WordPress/gutenberg/issues/47812